src/api/services/tagService.ts

```diff
@@ -8,11 +9,13 @@ import { tagFromDto } from '../mappers/tagMapper';
  * Fetches tags by given array of ids.
  * @param ids List of ids.
  */
-export const fetchTags = async(ids: number[]): Promise<Tag[]> => {
+export const fetchTags = async(tags: VisualNovelTag[]): Promise<Tag[]> => {
   const { data } = await http.post<TagDto[]>(
     ApiUrls.Tags,
```

Comment:
I think it's strange to have `tags` param to fetch tags, don't you think so? Need more context why you need this.

Reply:
Btw, update docs